lms/templates/header/navbar-authenticated.html

```diff
@@ -57,7 +58,11 @@
       </div>
     % endif
     <div class="mobile-nav-item hidden-mobile nav-item">
-      <a class="help-link" href="${get_online_help_info(online_help_token)['doc_url']}" target="_blank">${_("Help")}</a>
+      % if online_help_token == "instructor":
+        <a class="help-link" href="${get_online_help_info(online_help_token)['doc_url']}" target="_blank">${_("Help")}</a>
```

Comment:
Are we sure that only for instructor we want to redirect to doc_url? What about admin, support staff users?

Reply:
Yes right now we are supposed to add exception only for instructors. In future story we may define some more exceptions.

Reply:
@tasawernawaz To clarify, this is not instructor *permission* or *role*, correct? This actually checks for the specific Help token ``` instructor```. I don't believe there are admin or support Help tokens.

Reply:
thats true.